tgui/packages/tgui/interfaces/CommunicationsConsole.js

```diff
@@ -1,4 +1,5 @@
 import { sortBy } from "common/collections";
+import DOMPurify from 'dompurify';
 import { capitalize } from "common/string";
```

Comment:
```suggestion
import { capitalize } from "common/string";
import DOMPurify from 'dompurify';
```

please i made my imports beautifully sorted in alphabetical order :(

Reply:
sorry i will fix :(